src/components/SingleEvent/EventInfo.js

```diff
@@ -120,12 +121,7 @@ class EventInfo extends Component {
         </ContractAddress>
         <EventImage src={party.image || 'https://placeimg.com/640/480/tech'} />
         <Organiser>
-          <Avatar
-            src={`https://avatars.io/twitter/${getSocial(
-              party.owner.social,
-              'twitter'
-            ) || 'randomtwitter'}`}
-          />{' '}
+          <TwitterAvatar social={party.owner.social} avatar={Avatar}/>
```

Comment:
We shouldn't be passing the styled `Avatar` through. `TwitterAvatar` should import it and use it internally.

Reply:
Agreed